packs/equipment/robe-of-the-archmagi-greater.json

```diff
@@ -44,11 +44,58 @@
                 "key": "FlatModifier",
                 "label": "Robe of the Archmagi (against arcane spells)",
                 "predicate": [
-                    "arcane"
+                    "arcane",
+                    "self:caster:tradition:arcane"
                 ],
                 "selector": "saving-throw",
                 "type": "circumstance",
                 "value": 1
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "override",
+                "predicate": [
+                    {
+                        "not": "self:caster:tradition:arcane"
+                    }
+                ],
+                "relabel": "Robe of the Archmagi (worn by the unworthy)",
+                "selector": "saving-throw",
+                "slug": "resilient",
+                "value": 0
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "override",
+                "predicate": [
+                    {
+                        "not": "self:caster:tradition:arcane"
+                    }
+                ],
+                "relabel": "Robe of the Archmagi (worn by the unworthy)",
+                "selector": "ac",
+                "slug": "explorers-clothing",
+                "value": 0
```

Comment:
```suggestion
                "suppress": true
```

Reply:
I thought about that, but there's no indicate of why it's not applied.  There's already tons of people who don't get why their un-invested armor doesn't work.  The relabel makes it far more obvious why the AC bonus is 0.

Reply:
If you're keeping the labels, consider localizing them.

Reply:
I'd like to keep the labels because I think it's more user friendly.  But I guess @stwlam doesn't like the labels and wants them gone?  Or perhaps there's another interaction with the system that make suppress preferable.

I can localize.  I see "(against arcane spells)" could also be localized too.